src/YoutubeIframe.js

```diff
@@ -41,7 +52,7 @@ const YoutubeIframe = (props, ref) => {
     onError = _err => {},
     onReady = _event => {},
     playListStartIndex = 0,
-    initialPlayerParams = {},
+    initialPlayerParams,
     allowWebViewZoom = false,
     forceAndroidAutoplay = false,
     onChangeState = _event => {},
```

Comment:
I noticed default setting of these props are problematic because if not provided, they would always be new functions, resulting in unneccessary useEffect calls whose dependency arrays include them. Perhaps, we should fix it in a different PR.

Reply:
you're right, will have a look in a future PR